NOMENCLATURE.md

```diff
@@ -3,6 +3,12 @@ A list of terms used in the [CocoaPods](https://github.com/CocoaPods/CocoaPods)
 
 ## Glossary
 
+### Aggregate Target
+The target generated to aide the integration of the Pod targets with the user targets, prefixed with `Pods-`. Example: `Pods-MyApp.framework`
+
+### App Spec
+A Subspec which describes the details for creating an Application target, rather than a library or unit test target.
```

Comment:
"or test target" (as we now support ui and unit :))

Reply:
good call 👍